### PR TITLE
Fix some comment from #205 + fix bug and add sample/doc for the new option

### PR DESCRIPTION
**Pull request by jDramaix**

### CODE REVIEW

File: plugin/src/main/java/com/arcbees/chosen/client/resources/chozen.css [hunk 1]
```diff
@@ -414,3 +414,6 @@
 /*To make the gwt compiler happy*/
 .chzn-done{}
 }
+
+/* TODO JASON */
+.result-above{}
```
Comment (jDramaix) [inline plugin/src/main/java/com/arcbees/chosen/client/resources/chozen.css]: @jasonlemay

File: plugin/src/main/java/com/arcbees/chosen/client/resources/chozen.gss [hunk 2]
```diff
@@ -491,3 +491,8 @@
 
 .chzn-done {
 }
+
+/* TODO JASON */
```
Comment (jDramaix) [inline plugin/src/main/java/com/arcbees/chosen/client/resources/chozen.gss]: @jasonlemay

**Comment by meriouma** [comment]
LGTM

**Comment by christiangoudreau** [comment]
No test to update?

LGTM

**Comment by jDramaix** [comment]
> No test to update?

No

**Pull request merged**